### PR TITLE
Windows polish cleanup: default theme + dark popups + title sanitizer + configuration docs

### LINKED ISSUE #191: Windows: amux defaults to Tokyo Night while macOS picks up Ghostty theme — document and align config layering
## Summary

On macOS, amux shows the same theme the user has configured in Ghostty (because Ghostty is installed and \`theme_source = \"ghostty\"\` in the user's \`amux/config.toml\` picks it up). On Windows, amux falls back to the built-in Tokyo Night theme, because Ghostty isn't installed on most Windows boxes and the fallback path in \`startup.rs:305\` kicks in.

The user thought Tokyo Night had been removed from the codebase entirely — but it's still the hard-coded built-in fallback, and because Windows users don't have a Ghostty config, that fallback is what every fresh Windows install sees. This is a silent platform-specific presentation drift, not something the user knowingly opted into.

## How the theme layering actually works today

(Writing this down because it's not documented anywhere and the user was surprised by the behavior — part of the ask for this issue is to \*\*document this\*\* so future-us doesn't forget.)

### Config files

- **amux app config**:
  - macOS/Linux: \`~/.config/amux/config.toml\`
  - Windows: \`%APPDATA%\\amux\\config.toml\`
  - Parsed by \`crates/amux-core/src/config.rs:load_app_config\`
  - Schema: \`AppConfig\` struct in the same file
  - Relevant fields: \`theme_source\`, \`font_family\`, \`font_size\`, \`colors\` (overrides), \`shell\`, \`notifications\`, \`browser\`, \`keybindings\`

- **Ghostty config** (only read if \`theme_source = \"ghostty\"\`):
  - macOS: \`~/Library/Application Support/com.mitchellh.ghostty/config\`
  - Linux: \`~/.config/ghostty/config\`
  - Windows: \`%APPDATA%\\ghostty\\config\` (if the user has Ghostty installed on Windows at all)
  - Parsed by \`crates/amux-ghostty-config/src/lib.rs:GhosttyConfig::load\`

### Theme resolution (current code, \`crates/amux-app/src/startup.rs:293-316\`)

\`\`\`rust
let mut theme = match app_config.theme_source.as_str() {
    \"ghostty\" => {
        if let Some(ghostty_cfg) = amux_ghostty_config::GhosttyConfig::load() {
            // ... override font family/size from Ghostty config ...
            theme::Theme::from_ghostty(&ghostty_cfg)
        } else {
            tracing::warn!(\"theme_source = \\\"ghostty\\\" but no Ghostty config found, using default\");
            theme::Theme::default()   // <-- Tokyo Night lives here
        }
    }
    _ => theme::Theme::default(),      // <-- and here
};
theme.apply_color_config(&app_config.colors);  // [colors] section overrides on top
\`\`\`

So the built-in default **is** Tokyo Night (\`crates/amux-app/src/theme.rs:235\` \`// Tokyo Night\`), and you only escape it by:

1. Having \`theme_source = \"ghostty\"\` in \`amux/config.toml\` AND having a Ghostty config file present AND having that Ghostty config specify a different theme, OR
2. Using the \`[colors]\` section in \`amux/config.toml\` to override individual colors

## The actual bug

The platform split is unintentional. The user's \`amux/config.toml\` on Mac has \`theme_source = \"ghostty\"\`, so Mac gets whatever Ghostty's configured theme is. On a fresh Windows install with no \`amux/config.toml\` at all, \`theme_source\` defaults to \`\"default\"\` → Tokyo Night. And even if someone explicitly sets \`theme_source = \"ghostty\"\` on Windows, they probably don't have Ghostty installed, so the warning path fires and it falls back to Tokyo Night anyway.

Result: same binary, same config file layout, totally different presentation depending on platform. The user's Mac experience is not reproducible on Windows.

### Option A — Ship a better built-in default
Pick a single cross-platform dark theme (not Tokyo Night — user wants it gone) and make that the amux-branded default. Applies uniformly on all platforms when there's no user override. Effort: pick the palette, update \`theme::Theme::default()\`, done.

### Option B — Make Ghostty-config loading platform-aware
On Windows, probe additional plausible paths (e.g., if the user has Ghostty installed via winget), and fall through to the built-in default more gracefully.

### Option C — Ship amux with a curated set of themes the user can pick from
First-class theme picker in the UI, bundled palette library, user's chosen theme saved in \`amux/config.toml\`. Biggest polish win, bigger scope.

### Option D — Document the current behavior and do Option A
Ship the Option A default-palette change AND write \`docs/configuration.md\` (or similar) explaining the layering above so this doesn't silently drift again.

## Recommendation

**Option D**. The polish-parity goal is served by (1) picking a consistent cross-platform default and (2) documenting where configs live so platform-specific drift becomes visible in a git diff instead of surfacing months later as \"why is my Mac different from my VM?\". Option C is the right future direction but is a bigger scope.

Concrete deliverables for this issue:
- Replace \`theme::Theme::default()\` Tokyo Night palette with a chosen cross-platform default (separate design decision — worth a quick Mac screenshot comparison to pick something that looks good)
- Add \`docs/configuration.md\` documenting the config file locations, the \`theme_source\` field, the fallback chain, and the \`[colors]\` override mechanism
- Update \`CLAUDE.md\`'s Configuration section with a pointer to the new doc

## Severity

Medium for polish-parity. Doesn't break anything functionally, but it means Mac and Windows amux feel like two different products.

## Discovered during

Windows polish-parity follow-up to #186. User's quote: \"amux on windows is still using tokyo nights theme rather than the theme we're using on mac by default — i thought the tokyo nights theme was long gone.\"

### PR DESCRIPTION
**Pull request by daveowenatl**
## Summary

Bundled fast-follow addressing three open Windows-polish issues from the ARM64 VM testing session. 13 files, 764 insertions / 165 deletions, three new files:

- \`crates/amux-app/src/popup_theme.rs\` — cross-platform popup theming helper
- \`crates/amux-app/src/title_sanitize.rs\` — clean up ugly shell exe paths + 10 unit tests
- \`docs/configuration.md\` — full config reference

Closes #191, #198, #199.

## #198 — Sidebar workspace context menu was light-themed

Right-click on a workspace pill opened a white/light popup with dark text — egui's default theme, which clashes badly against amux's dark chrome. Same root cause as the menu bar popup bug from #196: egui reads **parent-ui style** to construct popup Frames, not the closure's ui.

Extracted \`MenuPalette\`, \`apply_menu_palette\`, and \`contrast_text\` out of \`menu_bar.rs\` (non-macOS gated) into a new cross-platform module \`popup_theme.rs\`. Applied at three points in \`sidebar.rs\`:

1. Parent \`ui\` before \`response.context_menu(...)\` — themes the top-level popup
2. Parent-of-submenu \`ui\` before \`ui.menu_button(\"Set Color\", ...)\` — themes the nested submenu
3. Inside the nested submenu closure — themes widgets rendered inside it

\`popup_theme\` is now the canonical location for this pattern — any future popup surface (tab bar context menus, find bar dropdowns, notification panel) can import and apply it consistently.

## #199 — Ugly shell-exe-path titles

ConPTY on Windows sets the initial window title of a spawned shell to the full absolute exe path:

> \`amux — C:\\Program Files\\WindowsApps\\Microsoft.PowerShell_7.6.0.0_arm64__8wekyb3d8bbwe\\pwsh.exe\`

New \`title_sanitize::sanitize_pane_title(raw) -> Cow<str>\` helper with these rules:

| Input                                                                           | Output   |
|---------------------------------------------------------------------------------|----------|
| \`\"\"\` or whitespace                                                          | \`\"?\"\`   |
| \`C:\\Program Files\\WindowsApps\\...\\pwsh.exe\`                               | \`pwsh\` |
| \`/bin/bash\`, \`/opt/homebrew/bin/fish\`                                       | \`bash\` / \`fish\` |
| \`pwsh.exe\`, \`cmd.exe\`, \`bash\`                                             | \`pwsh\` / \`cmd\` / \`bash\` |
| \`My Workspace\`, \`feat/my-branch\`, \`~/src/amux\`                            | unchanged (passthrough) |
| \`C:\\Users\\dave\\notes.md\` (non-shell absolute path)                         | unchanged |

10 unit tests cover every case plus case-insensitive extension matching and passthrough edge cases.

Applied at every user-facing \`pane.title()\` consumer:

- **\`managed_pane::title()\`** — the canonical dispatcher most UI goes through
- **Window title** (\`frame_update.rs\`) — switched from \`sf.pane.title()\` direct to \`managed.title()\`
- **Tab bar labels** (\`pane_render.rs:138\`) — sanitized inline before the \`user_title\` fallback
- **Rename modal pre-fill** (\`pane_render.rs:651\`) — so the modal opens with \`pwsh\` not the ugly path
- **Sidebar workspace display title** (\`notifications_ui.rs::active_surface_metadata\`)
- **Session save** (\`main.rs\`) — so restores show \`pwsh\` not the captured ConPTY default

**Deliberately not sanitized**: \`ipc_dispatch.rs::build_workspace_json\` — IPC output is consumed by external scripts that may want the raw shell-reported title.

## #191 — Replace Tokyo Night default + write \`docs/configuration.md\`

Replaced the hard-coded Tokyo Night palette in \`Theme::default()\` with a neutral amux-branded dark palette. Users who want Tokyo Night / Nord / Solarized / etc. should set \`theme_source = \"ghostty\"\` and configure their theme in Ghostty.

New default colors:
- Background: \`#14161E\`
- Foreground: \`#D0D4E0\`
- Accent: \`#3D7DFF\`
- ANSI palette: balanced saturation, readable dim + bright variants

Updated the \`config.rs\` doc comment for \`theme_source\` to stop naming Tokyo Night.

**Wrote \`docs/configuration.md\`**: the missing config reference covering every field, the three-layer theme resolution chain (built-in → \`theme_source\` → \`[colors]\`), keybinding syntax, \`menu_bar_style\` values, shell resolution behavior, notifications, browser, file locations, and a minimal working example. \`CLAUDE.md\`'s Configuration section now links to it.

## Test plan

- [x] \`cargo fmt --check\`, \`cargo clippy --workspace -- -D warnings\`, \`cargo test --workspace\` — all clean locally
- [ ] CI green on all three platforms
- [ ] ARM64 Windows VM verification:
    - [ ] Window title reads \`amux — pwsh\` (not the WindowsApps path)
    - [ ] Tab bar tab label reads \`pwsh\`
    - [ ] Right-click sidebar workspace → context menu renders in dark amux colors
    - [ ] Right-click → \`Set Color ▶\` → nested submenu also dark
    - [ ] Default theme looks neutral (not Tokyo Night)

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **New Features**
  * Centralized popup/modal/menu theming so context menus, submenus and modals use a consistent palette.
  * Title sanitization for panes applied broadly (tabs, renames, sessions, metadata) to simplify path-like/raw PTY titles.

* **Improvements**
  * Default visual theme updated to a neutral dark palette with a muted blue accent.
  * More consistent styling, contrast and hover behavior for menu bars, tab menus and related popups.

* **Documentation**
  * Added full configuration reference and clarified theme/menu config options.

### CODE REVIEW

File: crates/amux-app/src/title_sanitize.rs [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+/// `Cow::Owned` when it had to be transformed.
+pub(crate) fn sanitize_pane_title(raw: &str) -> Cow<'_, str> {
+    let trimmed = raw.trim();
+    if trimmed.is_empty() {
+        return Cow::Borrowed("?");
+    }
+
+    let looks_like_abs_path =
+        trimmed.contains('\\') || trimmed.contains('/') || has_drive_letter(trimmed);
+
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/title_sanitize.rs]: Fixed in e87bbb4. Tightened `looks_like_abs_path` to only match true absolute paths (leading `/`, `\\`, or drive-letter prefix with separator). Added tests for `tools/pwsh.exe`, `scripts\build\bash.sh`, and UNC paths.

File: docs/configuration.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+## `[notifications]` — Agent notification settings
+
+```toml
+[notifications]
+
+[notifications.sound]
+sound = "chime"   # built-in: "chime", "ding", "ping", "none"
+# or: sound = "/path/to/custom.wav"
+```
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply docs/configuration.md:150]: Fixed in e87bbb4. Rewrote the `[notifications]` section to match the real schema: `system_notifications`, `auto_reorder_workspaces`, `dock_badge`, `custom_command`, plus `[notifications.sound]` with `sound = "system"` / `"none"` / path and `play_when_focused`. Dropped the nonexistent named sounds.

File: docs/configuration.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+---
+
+## `[browser]` — In-app browser settings
+
+```toml
+[browser]
+search_engine              = "google"   # google, duckduckgo, bing, kagi, startpage
+open_terminal_links_in_app = false      # false → system browser, true → new browser tab
+user_agent                 = "Mozilla/5.0 ..."   # optional override
+download_dir               = "~/Downloads"        # optional; defaults to system Downloads
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply docs/configuration.md]: Fixed in e87bbb4. Changed the example to an absolute path and added a note that `~` is not expanded.

File: docs/configuration.md [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+
+## File locations (quick reference)
+
+| Purpose                  | Path                                                                        |
+|--------------------------|------------------------------------------------------------------------------|
+| amux main config         | `~/.config/amux/config.toml` (Unix) / `%APPDATA%\amux\config.toml` (Windows) |
+| amux session state       | `~/.config/amux/session.json` (Unix) / `%APPDATA%\amux\session.json` (Win)   |
+| Ghostty config (if used) | `~/.config/ghostty/config` (Unix) / `%APPDATA%\ghostty\config` (Windows)     |
+| Shell integration scripts| `~/.config/amux/shell/`                                                     |
+| Agent wrapper scripts    | `~/.config/amux/bin/`                                                       |
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply docs/configuration.md]: Fixed in e87bbb4. Expanded the file-locations table to show per-platform paths: session state now correctly points at `dirs::data_dir()` locations (`~/.local/share/amux/` on Linux, `~/Library/Application Support/amux/` on macOS, `%APPDATA%\amux\` on Windows).

File: crates/amux-app/src/sidebar.rs [hunk 5]
```diff
@@ -396,6 +396,19 @@ fn render_workspace_row(
...
+    // popup's background / border. Applying to the parent also
+    // propagates into the popup's child UI via inheritance so
+    // buttons and separators inside the menu are themed too.
+    //
+    // We also re-apply inside the nested `Set Color` submenu
+    // (which builds its own fresh Area + Frame from its local ui
+    // style) so that nested popup is themed as well.
+    let palette = crate::popup_theme::MenuPalette::from_theme(theme);
+    crate::popup_theme::apply_menu_palette(ui, palette);
     response.context_menu(|ui| {
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/sidebar.rs]: Fixed in e87bbb4. Removed the parent-`ui` `apply_menu_palette` call and wrapped `response.context_menu(...)` in `popup_theme::with_menu_palette`, which saves/applies/restores `ctx.style()` around the synchronous call so nothing leaks to subsequent sidebar rows. The nested `ui.menu_button("Set Color", ...)` still gets a scoped `apply_menu_palette(ui, palette)` inside the closure since `menu_button` reads parent-ui style (different egui code path).

File: crates/amux-app/src/pane_render.rs [hunk 6]
```diff
@@ -88,6 +88,19 @@ impl AmuxApp {
...
+        // default light theme. The tab-row painting that follows
+        // uses `painter` + explicit colors and ignores the style
+        // overrides this sets (buttons aren't used on the tab row
+        // itself, only inside the context menu popup).
+        {
+            let palette = crate::popup_theme::MenuPalette::from_theme(&self.theme);
+            crate::popup_theme::apply_menu_palette(ui, palette);
+        }
+
         {
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/pane_render.rs]: Fixed in e87bbb4. Removed the top-of-scope `apply_menu_palette(ui, palette)` block. The tab context menu now uses `popup_theme::with_menu_palette` scoped around the `tab_response.context_menu(...)` call, so nothing leaks into terminal overlays or modals.

File: crates/amux-app/src/pane_render.rs [hunk 7]
```diff
@@ -321,7 +342,13 @@ impl AmuxApp {
...
                 }
 
-                // Right-click context menu
+                // Right-click context menu. The palette was
+                // already applied to this `ui` at the top of the
+                // tab-bar scope (before the painter borrow), so
+                // the context menu popup inherits amux's chrome
+                // colors via the parent-ui style egui reads at
+                // Frame construction time. Same pattern as the
+                // sidebar workspace context menu.
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/pane_render.rs]: Fixed in e87bbb4. Comment updated to say `Response::context_menu` builds its popup Frame from `ui.ctx().style()` and references the `with_menu_palette` scope that themes it.

File: docs/configuration.md [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+- **Unix**: uses `$SHELL`, falling back to `/bin/bash`.
+- **Windows**: prefers `pwsh.exe` (PowerShell 7) if it's on `PATH`,
+  otherwise falls back to `$COMSPEC` (typically `cmd.exe`).
+
+---
+
+## `[colors]` — Palette overrides
+
+Apply specific color overrides on top of whichever theme source is active.
+Colors use hex notation (`"#rrggbb"` or `"#rrggbbaa"`).
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply docs/configuration.md]: Fixed in e87bbb4 — same edit as #3068877359 / #3068877359 above. Docs now state 6-digit `#rrggbb` only.

File: docs/configuration.md [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+new_tab       = "Ctrl+Shift+T"
+close_tab     = "Ctrl+Shift+W"
+split_right   = "Ctrl+Shift+D"
+split_down    = "Ctrl+Shift+E"
+# ...
+```
+
+See `crates/amux-core/src/config.rs` (`KeybindingsConfig::defaults_for_platform`)
+for the complete list of actions and their default bindings on each platform.
+
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply docs/configuration.md:202]: Fixed in e87bbb4. Docs now reference `KeybindingsConfig::platform_defaults()` and `resolved()` (the real function names) and point at the `Action` enum for the complete list.

File: crates/amux-app/src/frame_update.rs [hunk 10]
```diff
@@ -15,6 +15,29 @@ impl eframe::App for AmuxApp {
...
+        // (see `egui/src/menu.rs:392` / `Frame::menu(&button.ctx.style())`),
+        // not the parent ui's style. Per-call-site
+        // `apply_menu_palette(ui, palette)` has no effect on
+        // context menus.
+        //
+        // Done once per frame so theme changes (e.g. the user
+        // editing `theme_source` in the config) propagate on the
+        // next frame without a restart. The operation is cheap:
+        // one `Arc::make_mut` on the ctx style + a handful of
+        // field writes.
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/frame_update.rs]: Fixed in e87bbb4 — the whole ctx-level palette block was removed from `frame_update.rs` in this commit (see the response to #3068906362). Popup theming is now per-call-site via `with_menu_palette`, so there's no stale 'per-frame' comment to mislead anyone.

File: crates/amux-app/src/title_sanitize.rs [hunk 11]
```diff
@@ -0,0 +1,231 @@
...
+/// Returns the shell extension suffix (e.g. `".exe"`) if `s` ends
+/// with one from `SHELL_EXTENSIONS`. Matches case-insensitively.
+fn shell_extension(s: &str) -> Option<&'static str> {
+    let lower = s.to_ascii_lowercase();
+    SHELL_EXTENSIONS
+        .iter()
+        .find(|ext| lower.ends_with(*ext))
+        .copied()
+}
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/title_sanitize.rs]: Fixed in e87bbb4. Replaced the `to_ascii_lowercase()` allocations with `eq_ignore_ascii_case` for basename matching and an allocation-free `ends_with_ignore_ascii_case` helper for extension matching. The common passthrough case now doesn't allocate.

File: crates/amux-app/src/popup_theme.rs [hunk 12]
```diff
@@ -0,0 +1,210 @@
...
+//! `Frame::menu(ui.style())` in `egui::menu`). If the parent ui
+//! hasn't been themed, popups fall back to egui's default visuals —
+//! which is a light-mode look that clashes badly with amux's dark
+//! chrome.
+//!
+//! [`apply_menu_palette`] takes a [`MenuPalette`] (derived from the
+//! current theme) and mutates the ui's visuals so that any popup
+//! built afterwards picks up our colors. Call it on the PARENT ui
+//! BEFORE opening the popup, not inside the closure — by the time
+//! the closure runs, egui has already constructed the outer Frame.
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/popup_theme.rs]: Fixed in e87bbb4. Rewrote the top-level `popup_theme` doc comment: it now clearly separates the two cases — `ui.menu_button` / `popup_below_widget` read **parent ui** style, `Response::context_menu` reads **ctx** style — and points callers at `apply_menu_palette` vs. `with_menu_palette` accordingly.

File: crates/amux-app/src/frame_update.rs [hunk 13]
```diff
@@ -15,6 +15,29 @@ impl eframe::App for AmuxApp {
...
+        {
+            let palette = crate::popup_theme::MenuPalette::from_theme(&self.theme);
+            crate::popup_theme::apply_menu_palette_to_ctx(ctx, palette);
+        }
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/frame_update.rs]: Fixed in e87bbb4. Completely removed the unconditional `apply_menu_palette_to_ctx` call from `AmuxApp::update`. Popup theming is now scoped per-call-site via `popup_theme::with_menu_palette`, which saves the current ctx style, applies the menu palette, runs the closure (synchronously — `context_menu` builds its Frame during that call), then restores the original style. Non-menu widgets painted before/after no longer inherit transparent fills or missing borders. The `apply_menu_palette_to_ctx` function is now private and only used by the scoped helper.

File: crates/amux-app/src/theme.rs [hunk 14]
```diff
@@ -229,46 +230,68 @@ fn lighten_rgb(r: u8, g: u8, b: u8, amount: f32) -> Color32 {
...
 impl Default for Theme {
+    /// amux's built-in default theme: Monokai Classic — the same
+    /// palette that ships as cmux's default and that the author
+    /// uses via Ghostty on macOS. Windows and Linux users get the
+    /// same look out of the box without needing to install Ghostty
+    /// or write a config file; users who want something different
+    /// can set `theme_source = "ghostty"` to read their Ghostty
+    /// config instead, or override individual colors in the
+    /// `[colors]` section of `amux/config.toml`.
+    /// See `docs/configuration.md` for the full layering.
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/theme.rs:241]: Fixed in the commit immediately before the review batch (6a9cd7b) and kept consistent here — `Theme::default()` was deliberately switched to Monokai Classic for macOS parity. All doc comments (theme.rs, configuration.md, theme_source in config.rs) now agree it's Monokai Classic.

File: crates/amux-core/src/config.rs [hunk 15]
```diff
@@ -423,8 +423,13 @@ pub struct AppConfig {
...
     pub font_family: String,
-    /// Theme source: "default" uses built-in Tokyo Night, "ghostty" loads
-    /// colors/fonts from Ghostty's config file (`~/.config/ghostty/config`).
+    /// Theme source. `"default"` uses amux's built-in dark palette
+    /// (a neutral theme; see `crates/amux-app/src/theme.rs`);
+    /// `"ghostty"` loads colors and fonts from Ghostty's config
+    /// file at `~/.config/ghostty/config` (or the platform-
+    /// appropriate equivalent). Individual colors can still be
+    /// overridden in the `[colors]` section on top of either
+    /// source.
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-core/src/config.rs]: Fixed in e87bbb4. `AppConfig.theme_source` doc comment now says 'Monokai Classic — the same palette cmux ships as its default'.

File: crates/amux-app/src/title_sanitize.rs [hunk 16]
```diff
@@ -0,0 +1,231 @@
...
+/// Lowercase-compare `s` against [`KNOWN_SHELL_BASENAMES`].
+fn is_known_shell_basename(s: &str) -> bool {
+    let lower = s.to_ascii_lowercase();
+    KNOWN_SHELL_BASENAMES.iter().any(|b| *b == lower)
+}
+
+/// True if `s` starts with a Windows drive letter (`C:`, `d:`, ...).
+fn has_drive_letter(s: &str) -> bool {
+    let bytes = s.as_bytes();
+    bytes.len() >= 2 && bytes[0].is_ascii_alphabetic() && bytes[1] == b':'
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/title_sanitize.rs]: Fixed in e87bbb4. Renamed to `has_drive_letter_prefix` and now requires a `\` or `/` after the colon, so `C:pwsh.exe` is no longer treated as absolute. Added a test (`drive_relative_prefix_is_not_absolute`) to lock in the behavior.

File: docs/configuration.md [hunk 17]
```diff
@@ -0,0 +1,240 @@
...
+- **Windows**: prefers `pwsh.exe` (PowerShell 7) if it's on `PATH`,
+  otherwise falls back to `$COMSPEC` (typically `cmd.exe`).
+
+---
+
+## `[colors]` — Palette overrides
+
+Apply specific color overrides on top of whichever theme source is active.
+Colors use hex notation (`"#rrggbb"` or `"#rrggbbaa"`).
+
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply docs/configuration.md]: Fixed in e87bbb4. Updated `docs/configuration.md` to state only 6-digit `#rrggbb` is supported and noted that `ColorsConfig::parse_hex` is the authoritative parser.

File: crates/amux-app/src/title_sanitize.rs [hunk 18]
```diff
@@ -0,0 +1,231 @@
...
+//! from its default prompt function, but that depends on the
+//! user's profile actually running the default prompt AND on pwsh
+//! translating the title-set into an OSC sequence ConPTY can
+//! forward — which doesn't reliably happen on every Windows build
+//! we've tested.
+//!
+//! [`sanitize_pane_title`] is a best-effort cleanup applied at
+//! every title consumer (in practice, applied once at the bottom of
+//! `managed_pane::surface_title` so every consumer inherits the
+//! cleanup for free). Rules:
```
[the remark this answers is not in the document]
Comment (daveowenatl) [reply crates/amux-app/src/title_sanitize.rs]: Fixed in e87bbb4. Module doc now references `ManagedPane::title()` as the canonical call site.

**Comment by daveowenatl** [comment]
## Review body findings (no inline to reply to)

Addressed alongside the inline comments in e87bbb4.

**CodeRabbit review 4094453720 — nitpick on `docs/configuration.md:200-202`**
Fixed. The precedence section now spells out the full config path on each platform (`~/.config/amux/config.toml` on Linux, `~/Library/Application Support/amux/config.toml` on macOS, `%APPDATA%\amux\config.toml` on Windows) instead of the ambiguous short `amux/config.toml`.

**CodeRabbit review 4094476087 — nitpick on `popup_theme.rs:75-80`**
Fixed. `contrast_text` and the `hover_bg` brighten/darken branch now share a single `perceived_luma` helper using the same Rec.601 coefficients, so they can't disagree at color edges.
